test_req.py

```
@@ -0,0 +1,10 @@
+import requests
+
+try:
+    requests.get(
+        "https://nonexistent.maps.googleapis.com/maps/api/js",
+        params={"key": "SECRET_KEY_123"},
+        timeout=0.01,
+    )
+except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)